pyproject.toml

```diff
@@ -16,7 +16,6 @@ show_missing = true
 warn_unused_configs = true
 follow_imports = "skip"
 strict = true
-exclude = ['SSL\.py$']
 
 [[tool.mypy.overrides]]
 module = "OpenSSL.crypto"
```

Comment:
At this point these cover the entire module, can we collapse into a single block somehow?

Reply:
Indeed :)